x-pack/legacy/plugins/ml/public/components/data_recognizer/index.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-
-
 import './data_recognizer_directive';
 import './data_recognizer';
```

Comment:
I think this line can be removed, it's not angularjs which would need importing like this.

Reply:
Done in https://github.com/elastic/kibana/pull/45261/commits/67ca8ff26b35d595f64369919174634d7bcb9978